cytoflowgui/tests/test_tasbe.py

```diff
@@ -167,9 +160,8 @@ def testNotebook(self):
 
         nb_data = locals()['ex_2'].data
         remote_data = self.workflow.remote_eval("self.workflow[-1].result.data")
-        
-        self.assertTrue((nb_data == remote_data).all().all())
+        pd.testing.assert_frame_equal(nb_data, remote_data)
```

Comment:
Had to change to using this function for the testing -- it does a slightly fuzzy comparison: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.testing.assert_frame_equal.html